crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1890,15 +1913,18 @@ pub(crate) fn convert_to_validators(
                 pending_total_sui_withdraw: Some(BigInt::from(v.pending_total_sui_withdraw)),
                 pending_pool_token_withdraw: Some(BigInt::from(v.pending_pool_token_withdraw)),
                 voting_power: Some(v.voting_power),
-                // stake_units: todo!(),
+                // stake_units: todo!(), // TODO
                 gas_price: Some(BigInt::from(v.gas_price)),
                 commission_rate: Some(v.commission_rate),
                 next_epoch_stake: Some(BigInt::from(v.next_epoch_stake)),
                 next_epoch_gas_price: Some(BigInt::from(v.next_epoch_gas_price)),
                 next_epoch_commission_rate: Some(v.next_epoch_commission_rate),
                 at_risk,
                 report_records,
-                // apy: todo!(),
+                apy: apys.as_ref().and_then(|a| {
+                    let apy = a.iter().find(|x| x.address == v.sui_address);
+                    apy.map(|x| x.apy as u64) // apy is f64, but we expect an Int in graphql
```

Comment:
How to handle the conversion from f64 to u64, what do we want to store in the u64 field?